console_backend/src/shared_state.rs

```diff
@@ -680,9 +680,6 @@ impl SettingsTabState {
 // Navbar Types.
 
 /// Directory struct for storing informating and helpers pertaining to project directory.
-///
-/// Taken from swift-cli/swift/src/types.rs.
-/// impl taken from swift-cli/swift/src/lib.rs.
```

Comment:
remove comment that references internal code, and it may have drifted by now anyway